app/locations/none.js

```diff
@@ -0,0 +1,37 @@
+import Ember from 'ember';
+
+const {
+  computed: { reads },
+  inject: { service },
+  get
+} = Ember;
+
+const TEMPORARY_REDIRECT_CODE = 307;
```

Comment:
Agreed on `307 - TEMPORARY REDIRECT`. If we were trying to maintain HTTP 1.0 compatibility `302 - FOUND` would make sense. As it is, `307` is a very clear spec. We could make the code configurable with a `307` default, so people can override it. Would anyone object to that?